pkg/scheduler/api/helpers/helpers.go

```diff
@@ -43,6 +43,35 @@ func Min(l, r *api.Resource) *api.Resource {
 	return res
 }
 
+// Max is used to find the max of two resource types
```

Comment:
1. Suggest to make the it clear for the comments such as `Max returns the resource object with larger value in each dimension.`
2. Suggest to add another parameter `defaultValue` to fill the defalut value when dealing with the unequal dimensions. Perhaps the similar tool functions can be referred to in this file.